rocAL/rocAL/include/api/rocal_api_meta_data.h

```diff
@@ -61,7 +61,7 @@ extern "C" RocalMetaData ROCAL_API_CALL rocalCreateTFReaderDetection(RocalContex
 /// \param rocal_context
 /// \param source_path path to the coco json file
 /// \return RocalMetaData object, can be used to inquire about the rocal's output (processed) tensors
-extern "C" RocalMetaData ROCAL_API_CALL rocalCreateCOCOReader(RocalContext rocal_context, const char* source_path, bool is_output);
+extern "C" RocalMetaData ROCAL_API_CALL rocalCreateCOCOReader(RocalContext rocal_context, const char* source_path, bool is_output, bool mask);
```

Comment:
By default set mask to false

Reply:
Fiona added a new MetaDataType in latest commit and we are using that for checking which segmentation will be used for our pipeline

Reply:
make mask as const @SundarRajan28